.gitignore

```diff
@@ -39,3 +39,4 @@
 # Ignore Mac and Linux file system files
 *.swp
 .DS_Store
+.env*
```

Comment:
There was already a .env* in line 37, is there a reason why you have added it again?

Reply:
I didn't have it, I had to put my cloudinary key otherwise I couldn't load the app